openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -33,16 +35,64 @@
 ]
 
 
-def get_featured_course():
+def get_course_card_data(course, org_prefetched=False):
+    """
+    Get course data required for home page course card
+
+    :returns (dict): dict of course card data
+    """
+    pakx_short_logo = '/static/pakx/images/mooc/pX.png'
+    course_custom_setting = getattr(course, 'custom_settings', None)
```

Comment:
Suggestion: we should write a util function, which takes course & performs get_or_create functionality for custom_settings
WDYT?
This way we can reduce custom_settings null checks on multiple places.